web/themes/custom/novel/templates/layout/eventinstance--list-teaser.html.twig

```diff
@@ -7,7 +7,7 @@
   categories: content.event_categories,
   date: content.date,
   title: content.title,
-  description: content.event_description,
+  description: content.event_description|render|striptags('p'),
```

Comment:
I expect this to be a temporary fix until we land DDFFORM-298.

Reply:
I've removed it, as the PR is ready for DDFFORM-298 now.